config/database.yml

```diff
@@ -20,6 +20,7 @@ default: &default
   # For details on connection pooling, see Rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  url: <%= ENV.fetch("DATABASE_URL", "postgres://localhost:5432") %>
```

Comment:
Why was this needed?

Reply:
Previous setup assumed that everyone is running postgres as a system-level service. I'm running postgres via docker to handle multiple PG versions for different projects so I was unable to do anything without making this change

Reply:
I thought just setting the `DATABASE_URL` in your local env would be enough and it would get picked up automatically?